app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+
+        name = f"Name: {self.name}"
+        health = f"Health: {self.health}"
+        hidden = f"Hidden: {self.hidden}"
+
+        return f"{{{name}, {health}, {hidden}}}"
+
+
+class Herbivore(Animal):
+
+    def hide(self) -> bool:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+
+    def bite(self, prey: Animal) -> None:
+        if isinstance(prey, Herbivore) and prey.hidden is False:
+            prey.health -= 50
+            if prey.health <= 0:
+                self.alive.remove(prey)
```

Comment:
Read the checklist. You don't need self, make it static. 


![image](https://user-images.githubusercontent.com/104218550/218093614-5be22afa-4ccf-460a-9617-6d251ef3d5c2.png)

Reply:
Fixed.